src/content.js

```diff
@@ -134,16 +147,33 @@ async function buildMetaIndex(contentDir, invaderDefsDir, baseUrl, packageVersio
 }
 
 async function renderContent(metaIndex, outputDir) {
-  await Promise.all(metaIndex.pages.map(async (page) => {
+  const searchDocs = await Promise.all(metaIndex.pages.map(async (page) => {
     const templateName = page.template || "default";
     const renderTemplate = templates[templateName];
     if (!renderTemplate) {
       throw new Error(`The template '${templateName}' does not exist`);
     }
-    const result = renderTemplate(page, metaIndex);
+    const {htmlDoc, searchDoc} = renderTemplate(page, metaIndex);
     await fs.mkdir(path.join(outputDir, ...page._pathParts), {recursive: true});
-    await fs.writeFile(path.join(outputDir, ...page._pathParts, "index.html"), result, "utf8");
+    await fs.writeFile(path.join(outputDir, ...page._pathParts, "index.html"), htmlDoc, "utf8");
+    return searchDoc;
   }));
+
+  const searchIndex = new MiniSearch({
+    idField: "path",
+    fields: ["title", "text", "keywords"],
+    storeFields: ["title"],
+    processTerm: (term, _fieldName) => {
+      return STOP_WORDS.indexOf(term) == -1 ? term.toLowerCase() : null
```

Comment:
Shouldn't `term` in the `indexOf` here also be lowercased? It looks like we will miss stop words that have capital letters in them on the page.

Reply:
According to minisearch docs it normalizes all terms to lowercase already, but I’ll double check that.

Reply:
I don't mean that. I mean we're specifically excluding a few words here, but when filtering those words, we don't lowercase it beforehand. So, for ex: `Not` would end up being added as a potential search term while `not` would not.

By lowercasing it inside of the `indexOf` we would avoid `Not` `nOt` `NOT` all being added to the search terms even though we specifically want to avoid adding `not`

Reply:
Oh I see what you mean, derp. I'll lowercase the term before checking the stop words membership.